src/types/Status.ts

```diff
@@ -0,0 +1,5 @@
+export enum Status {
+  ALL = 'All',
+  ACTIVE = 'Active',
+  COMPLETED = 'Completed',
```

Comment:
Use camel case for enum
```suggestion
  All = 'All',
  Active = 'Active',
  Completed = 'Completed',
```